### PR TITLE
[VS Code] Add access token to cluster config

### PR DESCRIPTION
**Pull request by yiyione**

### CODE REVIEW

File: contrib/pai_vscode/package.json [hunk 1]
```diff
@@ -437,6 +438,7 @@
         "@types/request-promise-native": "^1.0.15",
         "@types/semver-compare": "^1.0.0",
         "@types/uuid": "^3.4.4",
+        "fast-glob": "^3.0.4",
```
Comment (sunqinzheng) [inline contrib/pai_vscode/package.json]: fast-glob is not used

File: contrib/pai_vscode/src/pai/paiJobManager.ts [hunk 2]
```diff
@@ -623,9 +623,11 @@ export class PAIJobManager extends Singleton {
                 const codeDir: string = path.join(taskDir, param.config.jobName);
                 await fs.ensureDir(codeDir);
                 if (param.upload) {
-                    // copy from local
+                    // copy from local`
```
Comment (sunqinzheng) [inline contrib/pai_vscode/src/pai/paiJobManager.ts]: typo

Comment (yiyione) [reply contrib/pai_vscode/src/pai/paiJobManager.ts]: Done.

File: contrib/pai_vscode/package.json (hunk 1 above)
Comment (yiyione) [reply contrib/pai_vscode/package.json]: Done.

**Pull request merged**